recipe/build.sh

```diff
@@ -2,6 +2,9 @@
 
 set -ex
 
+# This is used to detect if it's in the process of building pytorch
+export IN_PYTORCH_BUILD
```

Comment:
Are you sure you didn't need to assign it some value?

Reply:
CMake, in a boolean context (e.g., if($ENV{MY_ENV})), an empty string is considered false. Let me assign a specific value to fix it. Thanks for catching this!